popup.js

```diff
@@ -210,6 +228,8 @@ function findCard(shuffle = false) {
         }
         let target = cards[cardPositionIndex]
 
+        target.querySelector('ytd-thumbnail').style ='animation: 0.4s ease 0s infinite alternate none running jump;'
```

Comment:
```suggestion
        target.querySelector('ytd-thumbnail').style.animation ='0.4s ease 0s infinite alternate none running jump;'
```
more maintenable and lisible